Gruntfile.js

```diff
@@ -278,6 +278,10 @@ module.exports = function(grunt) {
       translations: {
         files: ['translations/*'],
         tasks: ['appcache', 'deploy']
+      },
+      node_modules: {
+        files: ['node_modules/**/*'],
```

Comment:
I just profiled it on my machine. Without watching node_modules grunt idles at about 2.3% CPU. With watching node_modules it's 100% CPU. I vote for reverting this change...